DuckDuckGo/TabViewController.swift

```diff
@@ -374,6 +385,8 @@ extension TabViewController: WebEventsDelegate {
     func webpageDidStartLoading() {
         Logger.log(items: "webpageLoading started:", Date().timeIntervalSince1970)
         resetSiteRating()
+        reloadScripts(with: siteRating!.protectionId)
```

Comment:
Is there any chance the SiteRating could be nil here? Consider a guar check.

Reply:
If the page has started loading then there must be a url and in that case there’s a site rating, but yep, can wrap in a guard in case that changes later.

Reply:
My concern with non-local force unwraps is that you lose the compiler check forever more and bugs can creep in as the code gets refactored. You'd be hard pressed to find a swift style guide that doesn't explicitly call it out. I hear what you're saying about the site rating definitely existing in this instance though. If you want to avoid a guard, you could refactor to grab the instance as it is created and pass it to this method as a non-optional param.

What you've got is technically correct, so I'm approving the PR but do think on it!

Reply:
Yep, I agree in this case. It’s wrong to assume the previous methods will behave the same way forever (especially when we don’t have unit tests). I’ll make the change before I merge.